### ISSUE TITLE
What does "ignored" mean in coverage summary?

### ISSUE THREAD

**Issue by rodoabad** [gotwarlost/istanbul#666]
Does it mean that it's been covered by other tests but there's no specific test for it?

**Comment by davglass** [comment]
No, ignored means that one of the istanbul comments was placed in the code and parts of the coverage collection was ignored.

https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md

**Comment by rodoabad** [comment]
That's when you explicitly ignoring things right?

```
=============================== Coverage summary ===============================
Statements   : 100% ( 11923/11923 ), 499 ignored
Branches     : 100% ( 3770/3770 ), 690 ignored
Functions    : 100% ( 2075/2075 ), 69 ignored
Lines        : 100% ( 3399/3399 )
================================================================================
```

I haven't ignored any yet I'm still getting some considered as ignored.

**Comment by gotwarlost** [comment]
Any transpilers in play? Babel for instance adds that comment.

**Comment by rodoabad** [comment]
@gotwarlost yes I require babel/register in my test since I'm using ES6 modules.

This is also how I setup my `npm` script.

`babel-node node_modules/isparta/bin/isparta cover --report html node_modules/mocha/bin/_mocha -- test/unit && istanbul check-coverage --statement 100 --branch 100 --function 100 --line 100`
